frontend/src/screens/settings/AutoUnlock.tsx

```diff
@@ -1,7 +1,6 @@
 import { AlertTriangle } from "lucide-react";
 import React from "react";
 
-import Container from "src/components/Container";
```

Comment:
wasn't the purpose of this so we do not have to specify the same size constraints in many places? why remove it here?

Reply:
containers are used at other places and along with size it also centralizes items. so for settings we don't need containers as now all the content should be left aligned